src/FSLibrary/MissionSimulatePubnet.fs

```diff
@@ -42,6 +42,116 @@ let simulatePubnet (context: MissionContext) =
                           }
                       )
                   )
+              loadGenOpCount = Some(context.loadGenOpCount |> Option.defaultValue (seq { 1 .. 100 }))
+              loadGenOpCountDistribution =
+                  Some(
+                      context.loadGenOpCountDistribution
+                      |> Option.defaultValue (
+                          // When no value is given, use the default values derived from observing the pubnet.
+                          seq {
+                              23963454
```

Comment:
I think it would be much cleaner if you placed these values into a file, and read from it here.

Reply:
Updated it!